### PR TITLE
[TM-5984] Maintain Assignments Remediation

### PR DESCRIPTION
**Pull request by jlam-mpc**
Changes:
- 

Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/988)

[TM-5984](https://metaphase.atlassian.net/browse/TM-5984)

[TM-5984]: https://metaphase.atlassian.net/browse/TM-5984?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/AssignmentsSeparations/Assignment/Assignment.jsx [hunk 1]
```diff
@@ -148,25 +155,30 @@ const Assignment = (props) => {
...
+      setWaiver(details?.WRT_CODE_RR_REPAY || 'N'); // Default to "Not Used"
       setSent(details?.NOTE_LAST_SENT_DATE || '');
     }
   }, [editMode]);
 
   const onSubmitForm = () => {
     const commonFields = {
+      tod_months_num: todMonths || null,
+      tod_other_text: todOther || null,
+      tod_adjust_months_num: null,
```
Comment (M00staff) [inline src/Components/AssignmentsSeparations/Assignment/Assignment.jsx:181]: this all looks familiar ... i think you have it right 🤔

i have some API code commented that says `# only custom/other TOD should have months and other_text` meaning, that the `tod_months` and `tod_other_text` values don't get sent to the frontend (in that specific section) UNLESS, the TOD is 'other,' and then the same the other way around (so only send months & other_text if TOD is 'other')

hope that makes sense lol - this looks right to me

**Review by M00staff — APPROVED** [review]
👍

File: src/Components/AssignmentsSeparations/Assignment/Assignment.jsx [hunk 2]
```diff
@@ -115,20 +121,19 @@ const Assignment = (props) => {
...
+      { 'Position Number': pos_results?.pos_num_text || NO_POSITION_NUMBER },
+      { 'Position Title': pos_results?.pos_title_desc || NO_POSITION_TITLE },
+      { 'Bureau': pos_results?.pos_bureau_short_desc || NO_BUREAU },
+      { 'Location': pos_results?.pos_location_code || NO_POST },
+      { 'Org': pos_results?.pos_org_short_desc || NO_ORG },
+      { 'Grade': pos_results?.pos_grade_code || NO_GRADE },
+      { 'Pay Plan': pos_results?.pos_pay_plan_code || NO_GRADE },
+    ];
+    /* eslint-enable quote-props */
+  }
```
Comment (scott062) [inline src/Components/AssignmentsSeparations/Assignment/Assignment.jsx:136]: I originally had this in the useEffect bc it wouldn't render changes on different pos num searches. It works fine without it now, did I miss something from an earlier change that would track pos_results changes? Or is it automatic re-renders from useSelector and I'm just dumb?

**Comment by scott062** [comment]
Noticed Disabled inputs - is that aligned with FSBID currently?

**Pull request merged**